docs/design/specs/PortablePdb-Metadata.md

```diff
@@ -523,3 +525,69 @@ Parent: Module
+
+##### <a name="CompilationMetadataReferences"></a>Compilation Metadata References (C# and VB compilers)
+Parent: Module
+
+Kind: {7E4D4708-096E-4C5C-AEDA-CB10BA6A740D}
+
+Stores information about all metadata references used to compile the module.
+
+The blob has the following structure:
+
+    Blob ::= MetadataReferenceInfo+
+    MetadataReferenceInfo ::= file-name aliases flags time-stamp file-size mvid
+
+| terminal   | encoding                                  | description                                               |
+|:-----------|:------------------------------------------|:----------------------------------------------------------|
+| file-name  | UTF8 NIL-terminated                       | Name of the metadata file (includes an extension).        |
+| aliases    | UTF8 NIL-terminated, comma-separated list | List of external aliases for the reference. May be empty. |
+| flags      | byte                                      | Flags.                                                    |
+| time-stamp | uint32                                    | PE COFF header Timestamp field.                           |
+| file-size  | uint32                                    | PE COFF header SizeOfImage field.                         |
+| mvid       | GUID (16 bytes)                           | Module Version Id (ModuleDef table field).                |
+
+The meaning of the _flags_ byte:
+
+| flag      | description                                                     |
+|:----------|:----------------------------------------------------------------|
+| 0b0000001 | Embed interop types.                                            |
+| 0b0000010 | The referenced file is an assembly (as opposed to a netmodule). |
+
+The remaining bits are reserved for future use and have currently no meaning.
+
+> The data can be used to find the reference in a file indexing service such as a symbol server. 
+> For example, the [Simple Symbol Query Protocol](https://github.com/dotnet/symstore/blob/master/docs/specs/Simple_Symbol_Query_Protocol.md) uses a combination of _file-name_, _time-stamp_ and _file-size_ as a [key](https://github.com/dotnet/symstore/blob/master/docs/specs/SSQP_Key_Conventions.md#pe-timestamp-filesize).
+> Other services might use the MVID as it uniquely identifies the module.
+
+##### <a name="CompilationOptions"></a>Compilation Options (C# and VB compilers)
+Parent: Module
+
+Kind: {B5FEEC05-8CD0-4A83-96DA-466284BB4BD8}
+
+Stores compilation options used to compile the module. Only captures information that is not present elsewhere in the PDB, in the PE headers or metadata of the module.
+
+The blob has the following structure:
+
+    Blob ::= (name value)*
+
+| terminal   | encoding            | description                      |
+|:-----------|:--------------------|:---------------------------------|
+| name       | UTF8 NIL-terminated | Name of the compilation option.  |
```

Comment:
Can there be duplicate names in this sequence?

Reply:
Good question - I'd say no. Will update.